src/classes/RD_RecurringDonations_TDTM.cls

```diff
@@ -86,6 +86,11 @@ public class RD_RecurringDonations_TDTM extends TDTM_Runnable {
                      r.npe03__Open_Ended_Status__c != system.label.npe03.RecurringDonationClosedStatus)) {
                     r.addError(system.label.npe03.RecurringDonationTooManyInstallmentsError);
                 }            
+                // Reset this field to false if the installment period is not Monthly or Quarterly
```

Comment:
@judisohn  Funny. I had this working (without realizing it) and then added logic to uncheck the "Always use last day of month" field when the installment period was not monthly or quarterly - which broke the use of that field for custom installment periods. Works now.